microsoft/binskim
# Add tests when investigation if we have the telemetry working for .net and Rust

During investigation if we have the telemetry working for .net and Rust,
I notice ther is no enough tests for it,
to test I have to add some test files,
since i have those created, I think we can check them it to save time for other dev testing the code.
Please check comments.
